docs/src/embeddings/embedding_functions.md

```diff
@@ -166,4 +166,4 @@ rs[2].image
 ![](../assets/dog_clip_output.png)
 
 Now that you have the basic idea about LanceDB embedding functions and the embedding function registry,
-let's dive deeper into defining your own [custom functions](./custom_embedding_function.md).
\ No newline at end of file
+let's dive deeper into defining your own [custom functions](./custom_embedding_function.md).
```

Comment:
Not sure why this line is showing a difference?

Reply:
trailing space was removed. sounds like automatically by your IDE. Thanks for cleaning that up tho :)